packages/clay-data-provider/src/index.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import React, {useState} from 'react';
+import {IDataProvider, NetworkStatus} from './types';
+import {useResource} from './useResource';
+
+interface IChildrenProps {
+	data: any;
+	error: string | boolean;
+	loading: boolean;
+	networkStatus?: NetworkStatus;
+	refetch: Function;
+}
+
+interface Props extends IDataProvider {
+	children: (data: IChildrenProps) => React.ReactElement;
```

Comment:
Even though this prop is an internal react one, I think it might be helpful to add a JSDoc to explain why its declared here, whether that be because you need a certain type of child or if you want to ensure its required.

Reply:
Yeah, I'll take care of adding that.